src/wg/format/png/reader.d

```diff
@@ -5,7 +5,9 @@ import wg.format.png.types;
 import wg.util.util: Result, asArrayOf;
 import wg.image.imagebuffer;
 
-// TODO: 
+import etc.c.zlib : inflateInit, inflate, inflateEnd, z_stream, z_streamp;
+
+// TODO:
 // 1. Make CommonMetadata using pHYs chunk data
 // 2. Make PngChunks smarter so they keep their allocator and chunks can easily be added and removed
 // 3. Check sRGB, gAMA and cHRM chunks and return the proper format string using them
```

Comment:
I just realized the third TODO item here is actually done so it can be deleted.

Reply:
Feels a bit unrelated but if you say so..?